packages/management/tkr-service/bundle/config/values.yaml

```diff
@@ -0,0 +1,6 @@
+#@data/values
+#@overlay/match-child-defaults missing_ok=True
+---
+tkrService:
+  namespace: tkr-system
+  createNamespace: true
```

Comment:
Based on the discussion offline, the namespace should be a pre-requisite, so must already exist.

Reply:
updated.